### DISCUSSION TITLE
Logging Level - Critical

### DISCUSSION THREAD

**Discussion by trile757** [rabbitmq/rabbitmq-server#11836] [Other] [answered]
Using Rabbit version 3.13.1-1.el8

I have two nodes running RabbitMQ. We want to set the log level at Critical. It has been set. However, I can't seem to find a way to "break" RabbitMQ to force a critical message in the rabbit.log. I was just wondering if anyone has an idea of what constitutes a critical alert. I stopped services on one node while tailing the rabbit.log on the other node. It only reported that it lost connection to its peer as a "warning".

**Comment by michaelklishin** [comment, marked as answer]
That level is not used by RabbitMQ. `rabbitmqctl eval` can be used to log a message at any level:

``` shell
rabbitmqctl eval 'rabbit_log:critical("Severity = critical").'

rabbitmqctl eval 'rabbit_log:error("Severity = error").'

rabbitmqctl eval 'rabbit_log:warning("Severity = warning").'
```

I highly recommend against disabling logging of errors, you are guaranteed to regret this decision. Use the `error` [log level](https://www.rabbitmq.com/docs/logging#log-levels) or better yet, leave it at `warning`.

**Comment by michaelklishin** [comment]
Setting log level to `critical` will effectively disable logging. You will not get any help from the core team without logs, so if you choose to do it (I guess to save a bit of disk space?), you are entirely on your own.

If the goal is to save disk space, a much better idea would be to [set up logrotate](https://www.redhat.com/sysadmin/setting-logrotate).

Also, `3.13.1` is five patch releases behind.
